File: Modul 4/Python/PRAK403-2310817220025-FirdaKhoirunisa.py
satu, dua = input().split()
x = int(satu)
y = int(dua)

if x > y:
    while x >= int(dua) and y <= int(satu):
        print(x, end=' ')
        print(y, end=' ')
        x -= 1
        y += 1
        if x == int(dua) - 1:
            break
        else:
            print("-", end=' ')
elif x < y:
    while x <= int(dua) and y >= int(satu):
        print(x, end=' ')
        print(y, end=' ')
        x += 1
        y -= 1
        if y == int(satu) - 1:
            break
        else:
            print("-", end=' ')

File: Modul 4/C/PRAK403-2310817220025-FirdaKhoirunisa.c
#include <stdio.h>

int main(){
    int satu, dua, x, y;
    scanf("%d %d", &satu, &dua);
    x = satu;
    y = dua;
    if (satu < dua){
        for(x = satu, y = dua; x <= dua && y >= satu; x++, y--){
           printf("%d %d", x, y);
           if (x == dua){
            break;
           }else{printf(" - ");}
        }    
    }
    if (satu > dua){
        for(x = satu, y = dua; x >= dua && y <= satu; x--, y++){
           printf("%d %d", x, y);
           if (x == dua){
            break;
           }else{printf(" - ");}
        }
    }return 0; 
}